utils/httpstream.go

```diff
@@ -44,17 +45,38 @@ func StartHTTPServer(handler Handler, path, format string) {
 	}
 
```

Comment:
I think we should add a runtime.LockOsThread() and defer runtime.UnlockOsThread() here...

Reply:
this doesn't return until the container exits, you're only worried about create->chmod->symlink ?

Reply:
https://github.com/fnproject/fdk-go/pull/31/commits/0a758a849edb933722bd40149b23dd1a4eab0382

Reply:
we could lock it around listen but meh

Reply:
I'm being overly paranoid.